src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+);
+
+Sum.defaultProps = {
```

Comment:
good! also, you can add default value as you did it in functions

`export function Sum({ a = 0, b = 0}) {...}`